cmd/builder/internal/command.go

```diff
@@ -83,10 +83,8 @@ func Command() (*cobra.Command, error) {
 
 func initConfig() error {
 	cfg.Logger.Info("OpenTelemetry Collector distribution builder", zap.String("version", version), zap.String("date", date))
-	// use the default path if there is no config file being specified
-	if cfgFile == "" {
-		cfgFile = "$HOME/.otelcol-builder.yaml"
```

Comment:
Interesting, I wonder why we had this here... But if it works, I'm glad to have this code removed.